python/refactor/nn/metrics/metrics_evaluator.py

```diff
@@ -0,0 +1,82 @@
+import torch as th
+from refactor.utils import get_scalar
+
+# we hide the implementation of evaluator from KGEModel
+# as one class should do one thing and fit() in KGEModel should not take responsibility
+# to aggregate metric results
+class MetricsEvaluator:
+    def __init__(self):
+        pass
+
+    def evaluate(self, results, data, graph):
+        # different models might have different results. Thus results might contain different number
+        # of elements to evaluate performance
+        pass
+
+class KGEMetricsEvaluator(MetricsEvaluator):
```

Comment:
It is better to call it RankingMetricsEvaluator. The method is not KGE specific.